nixnet_examples/can_frame_stream_io.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pprint
+import six
+import time
+
+from nixnet import constants
+from nixnet import nx
+from nixnet import types
+
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+def main():
+    database_name = ':memory:'
+    cluster_name = ''
+    list = ''
+    interface1 = 'CAN1'
+    interface2 = 'CAN2'
+    input_mode = constants.CreateSessionMode.FRAME_IN_STREAM
+    output_mode = constants.CreateSessionMode.FRAME_OUT_STREAM
+
+    with nx.Session(database_name, cluster_name, list, interface1, input_mode) as input_session:
+        print('Input session created successfully.')
+        with nx.Session(database_name, cluster_name, list, interface2, output_mode) as output_session:
+            print('Output session created successfully.')
+
+            print('Are you using a terminated cable? Enter Y or N')
+            terminated_cable = six.input()
+            if terminated_cable.lower() == "y":
+                output_session.intf_can_term = constants.CanTerm.OFF
+            else:
+                input_session.intf_can_term = constants.CanTerm.ON
+                output_session.intf_can_term = constants.CanTerm.ON
+
```

Comment:
We should explicitly set termination for both interfaces for clarity. I.e 
if terminated_cable.lower() == "y":
 output_session.intf_can_term = constants.CanTerm.OFF
input_session.intf_can_term = constants.CanTerm.ON